srml/sudo/src/lib.rs

```diff
@@ -14,8 +14,112 @@
 // You should have received a copy of the GNU General Public License
 // along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
 
-//! The Example: A simple example of a runtime module demonstrating
-//! concepts, APIs and structures common to most runtime modules.
+//! # Sudo Module
+//!
+//! ## Overview
+//! 
+//! The Sudo module allows for a single account (called the "sudo key")
+//! to execute dispatchable functions that require a `Root` call
+//! or designate a new account to replace them as the sudo key.
+//! Only one account can be the sudo key at a time.
+//! 
+//! You can start using the Sudo module by implementing the sudo [`Trait`].
```

Comment:
What's the point of this line? It's the same in every module and doesn't tell the user anything new about sudo.

Reply:
I think the motivation of this line is to link to the Trait, which is unique to sudo. Perhaps a better way to handle these redundancies is to start with a list? E.g.:

- link to Trait
- link to Call enum
- link to Module struct

Reply:
are these not all linked in the sidebar anyway?